bootstrap-app/src/cmr/bootstrap/api/routes.clj

```diff
@@ -130,6 +131,12 @@
             (= (keyword cache-name) provider-cache/cache-key)
             (provider-cache/refresh-provider-cache request-context)
 
+            (= (keyword cache-name) coll-for-gran-acls-caches/coll-by-concept-id-cache-key)
+            (coll-for-gran-acls-caches/refresh-entire-cache request-context)
+
+            (= (keyword cache-name) coll-for-gran-acls-caches/coll-by-provider-id-and-entry-title-cache-key)
+            (coll-for-gran-acls-caches/refresh-entire-cache request-context)
```

Comment:
Since this is 1 refresh call how about 
(or (= (keyword cache-name) coll-for-gran-acls-caches/coll-by-concept-id-cache-key)
      (= (keyword cache-name) coll-for-gran-acls-caches/coll-by-provider-id-and-entry-title-cache-key))
(coll-for-gran-acls-caches/refresh-entire-cache request-context)

Reply:
updated